floating-ip.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+/**
+ * Create floating IP actions endpoint
+ * @param  {Integer} ip The floating IP in question
+ * @return {String}     The relevent volume URL
+ */
+function getActions(ip) {
+	return `floating_ips/${ip}/actions`;
+}
+
+module.exports = {
+	listFloatingIPs: function () {
+		const val = 'floating_ips';
+		return this.request(val, {val});
+	},
+
+	createFloatingIP: function ({dropletId, region}) {
```

Comment:
Because this module is supporting Node 4, we can't destructure this param. Let's do something like this instead:

```js
createFloatingIP: function (body) {
  if (body.droplet_id && body.region) {
    throw new Error('...');
  }

  if (!body.droplet_id && !body.region) {
    throw new Error('...');
  }

  return this.request('floating_ips', {method: 'POST', val: 'floating_ip', { body }});
}
```